lib/diagnostic.js

```diff
@@ -39,20 +44,34 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name:  "Penelope",
+  species: "cat",
+  breed:  "Maine Coon",
+  vocalization: "meow",
+  Vocalize: function {
+    this.vocalization ++1
```

Comment:
We don't want to increment `vocalization` here, we simply want to return it. Something like this: `function() {return this.vocalization}` so that when we call `pet.Vocalize()`, "meow" is returned.